src/tools/homebrew/Formula/flow-php.rb

```diff
@@ -0,0 +1,16 @@
+class FlowPhp < Formula
+  desc "Flow is a first and the most advanced PHP ETL framework"
+  homepage "https://github.com/flow-php/flow"
+  url "https://github.com/flow-php/flow/releases/download/0.5.1/flow.phar"
+  sha256 "a96caaccf4382183edbd8a43e41b4c6307bba717867480aa69711ade7bca7204"
+
+  depends_on "php" if MacOS.version <= :el_capitan
```

Comment:
Why does it depend on php only when a version is lower or equal to el_capitan?

Reply:
Outdated, should be always php.